Kenneth-Macharia/FastFoodFast
# #163191835 tests setup + add GET ALL menus tests

#### What does this PR do?
- Adds automated tests for the GET ALL menus admin endpoint
#### Description of Tasks to be completed?
- Update requirement.txt
- Set up travis to run automated tests
- Develop and set up tests
- Refactor app code to ensure tests run efficiently and pass
#### How should this be manually tested?
- Install PostgreSQL on a local machine
- Create a database on that machine
- Create a local user to the database with full rights to it, secured by a password
- Clone this repo to the local machine with the database above
- Navigate to the root of the repo: '/FastFoodFast
- Check out the branch: 'ch-admin-menu-tests-163191835'
- Create a virtual environment: virtualenv <your_environment_name>
- Activate the virtual environment and install all app requirements: pip install -r requirements.txt
- Set the environment variables specified in the .env file with your details
- Set  the app environment variable : APP_SETTINGS=development
- Run tests on the app code: pytest --cov=app/api_v1 for test results as per below image:
#### What are the relevant pivotal tracker stories?
#163191835
![test](https://user-images.githubusercontent.com/42747960/51729738-f8e4e380-2085-11e9-96d8-6fa7321c2023.JPG)
